File: test-fixtures/cross-language/python/src/orphan.py
"""Dead-weight: never imported by any other file"""


def orphan_function(input_str: str) -> str:
    return input_str.upper()


class OrphanClass:
    def __init__(self, name: str, value: int) -> None:
        self.name = name
        self.value = value

    def process(self) -> int:
        return self.value * 2

File: test-fixtures/cross-language/typescript/src/orphan.ts
// Dead-weight: never imported by any other file
export function orphanFunction(input: string): string {
  return input.toUpperCase();
}

export class OrphanClass {
  name: string;
  value: number;

  constructor(name: string, value: number) {
    this.name = name;
    this.value = value;
  }

  process(): number {
    return this.value * 2;
  }
}
